src/pages/model/constructor/constructor-page/constructor-page.js

```diff
@@ -37,6 +37,10 @@ import { selectConstructorMethodAndMaterials } from '../../../../redux/selectors
 import { getPatterns } from '../../../../redux/pattern/pattern.actions';
 import { getMaterials } from '../../../../redux/material/material.actions';
 import LoadingBar from '../../../../components/loading-bar';
+import {
+  listItemImages,
+  isListItemAvailable
+} from '../../../../utils/constructor-page';
 
 const map = require('lodash/map');
 
```

Comment:
please, use import , not require and put it to our external imports

Reply:
fixed